http/query_handler.go

```diff
@@ -495,3 +504,45 @@ func SimpleQuery(addr, flux, org, token string) ([]byte, error) {
 	defer res.Body.Close()
 	return ioutil.ReadAll(res.Body)
 }
+
+func QueryHealthCheck(url string) check.Response {
+	u, err := newURL(url, "/health")
+	if err != nil {
+		return check.Response{
+			Name:    "query health",
+			Status:  check.StatusFail,
+			Message: errors.Wrap(err, "could not form URL").Error(),
+		}
+	}
+
+	insecureSkipVerify := true
```

Comment:
Hardcoding insecureSkipVerify to true seems a little suspect. Would it be better off as an explicit parameter to `QueryHealthCheck`? Off the top of my head I'm not sure how that gets plumbed through all the layers of query services.

Reply:
You are right.  All of the callers have `insecureSkipVerify` as a member, so I added it as a parameter to `QueryHealthCheck`.